app/repositories.php

```diff
@@ -14,13 +14,22 @@
 return static function (ContainerBuilder $containerBuilder) {
     $containerBuilder->addDefinitions([
         PersonRepository::class => static function (ContainerInterface $c): PersonRepository {
+            /** @var PersonRepository $repo */
```

Comment:
this is more verbose than I would like. I think it's fine to do this for now, but I can easily make it so we don't have to keep writing this by adding a simple stub file for ContainerInterface.